strings-csharp/ReadStringFromResourceFile/ReadStringFromResourceFile/Classes/ResourceItem.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace ReadStringFromResourceFile;
+public sealed class ResourceItem<T>
```

Comment:
I don't think we need this file at all for this article. Adding generics over-complicates it and we should focus on getting data from the resource file in string formats.